.github/workflows/release-please.yml

```diff
@@ -98,3 +98,29 @@ jobs:
             env:
                 MASTODON_ACCESS_TOKEN: ${{ secrets.MASTODON_ACCESS_TOKEN }}
                 MASTODON_HOST: ${{ secrets.MASTODON_HOST }}
+
+          #-----------------------------------------------------------------------------
```

Comment:
This needs to go first because Espree depends on `eslint-visitor-keys`

Reply:
Updated 👍🏻